### PR TITLE
REAL-124 - Security Vulnerabilities

### PR DESCRIPTION
**Pull request by JacksonKolb94**
Bumped The Versions. EZ GG NO RE

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,11 +9,11 @@ oauth2client==3.0.0
 protobuf==3.7.0rc3
 pyasn1==0.4.5
 pyasn1-modules==0.2.4
-pycryptodome==3.4.3
+pycryptodome>=3.6.6
```
Comment (paugajt) [inline requirements.txt]: Why the fuck are we doing this?!?

File: requirements.txt [hunk 2]
```diff
@@ -9,11 +9,11 @@ oauth2client==3.0.0
...
 Pyrebase==3.0.27
 python-jwt==2.0.1
 python-mimeparse==1.6.0
-requests==2.11.1
+requests>=2.20.0
```
Comment (paugajt) [inline requirements.txt]: You should ask for a refund on your CS degree

File: requirements.txt (hunk 1 above)
Comment (JacksonKolb94) [reply requirements.txt]: I don't know. I was drunk.

File: requirements.txt (hunk 2 above)
Comment (JacksonKolb94) [reply requirements.txt]: I did already. Something about, "you're a societal wheel now, Mr. Kolb. Get used to debt."

**Comment by JacksonKolb94** [comment]
+10 ez pz lemon squeezy

**Pull request merged**